R/validations.R

```diff
@@ -73,11 +74,12 @@ validate_has_data <- function(x,
 
 #' Validate that dataset has unique rows for key variables
 #'
-#' @description `r lifecycle::badge("stable")`
-#' @param x a data.frame
-#' @param key a vector of ID variables from `x` that identify unique records
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```

Reply:
https://github.com/insightsengineering/teal/pull/1052/files#r1462868213